src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -105,6 +120,21 @@ internal partial class BatchFixAllProvider : FixAllProvider
             return fixesBag.ToImmutableArray();
         }
 
+        private async Task AddDocumentFixesAsync(
+            Document document, ImmutableArray<Diagnostic> diagnostics,
+            ConcurrentBag<(Diagnostic diagnostic, CodeAction action)> fixes,
+            FixAllState fixAllState, IProgressTracker progressTracker, CancellationToken cancellationToken)
+        {
+            try
+            {
+                await this.AddDocumentFixesAsync(document, diagnostics, fixes, fixAllState, cancellationToken).ConfigureAwait(false);
+            }
+            finally
+            {
+                progressTracker.ItemCompleted();
+            }
```

Comment:
This pattern has been added a fair few times in this PR, might be worth an extension method to make it easier, so the caller of this method could just do something like:

```cs
using (var _ = progressTracker.ProcessItem())
{
	await this.AdddocumentFixesAsync(document, diagnostics, fixes, fixAllState, cancellationToken).ConfigureAwait(false);
}
```